projects/aws/bottlerocket-bootstrap/pkg/utils/userdata.go

```diff
@@ -24,28 +27,89 @@ type WriteFile struct {
 	Content     string
 }
 
+type UserDataSource struct {
+	Provider string
+	Prefix   string
+	Chunks   int
+}
+
+type ExternalUserData struct {
+	UserDataType   string         `yaml:"user_data_type"`
```

Comment:
This looks like this will always be "external" right? Should we get rid of this?